### ISSUE TITLE
What are the flags for the variant art so we can trigger changes with Monk's Active Tile Triggers

### ISSUE THREAD

**Issue by stevenagl12** [Aedif/TokenVariants#155]
Alright, possible a dumb question, but what are the token flags for the variant art images that you have selected for the image so that you can assign them to a Monk's Active Tile Trigger and be able to change their appearance automatically?

**Comment by Aedif** [comment]
What TVA images are you talking about? Shared (green arrows)?

**Comment by stevenagl12** [comment]
Yes, when you have assigned the variants for an actor, is there an internal
flag to be able to change them? Basically I am trying to be able to cause
mimic tokens to switch to an active state, or changelings to release their
true form on a button click.

**Comment by stevenagl12** [comment]
I'm trying to use a monk's active tile trigger to alter the variant image
with a single clock so I can do 20 tokens at once, instead of individually
changing each.

**Comment by IsaCorreia** [comment]
I'm trying to do something similar, but with the Eladrin Seasons.

Here's what I got (it's not working):
I wrote a macro that creates/updates a flag inside `token-variants` namespace.

```js
// Updates the flag "eladrinSeason" inside "token-variants" module namespace
const actorId = "4VQ2iNTYhOymlRvw";
var actor = game.actors.get(actorId);
var seasons = ["Outono", "Inverno", "Primavera", "Verao"];

var currEladrinSeason =  seasons[Math.floor(Math.random() * 4)];

const updateData = {
  ['flags.token-variants.eladrinSeason']: currEladrinSeason
};

actor.update(updateData);
```
(I'll add an image of said flag here later)

Then, I setup the token images
![image](https://github.com/Aedif/TokenVariants/assets/93007708/ecc1215b-a48e-49f1-97e5-fe4cf56fa9a7)

And lastly I used the "Mappings" option for the token to create conditions based on that flag I created.
![image-1](https://github.com/Aedif/TokenVariants/assets/93007708/4392aed0-ea50-4c7c-8433-1e8715cd101f)

The thing is, the expressions for Mapping don't seem to work properly. I tried adding script logs for when they are met or not, and they did run immediatly after I clicked "save & close", and ran 11 times.
I was testing with just two tokens, and it was supposed to show Token A, and it showed Token B, I was using the example of "hp>50%" and "hp<=50%" for this test.
But then it stops, and doesn't check for later updates on said flag anymore.

The string comparisson also doesn't work, even though the module's tutorial uses a similar expression in its tutorial 
![Screenshot_20240706_125926_Chrome](https://github.com/Aedif/TokenVariants/assets/93007708/5e69c867-0d06-4508-ad73-588d32a16f8d)

I suppose working with JS inside the Expression could be a solution, but I haven't found any mentions about what is the expected return for that in de docs.
